tests/test_qubit_device.py

```diff
@@ -21,7 +21,6 @@
 import pennylane as qml
 from pennylane import QubitDevice, DeviceError
 from pennylane.qnodes import QuantumFunctionError
-from pennylane import expval, var, sample
```

Comment:
Just for my understanding, how did these end up being not needed? Or were they extras also before?

Reply:
If I remember correctly, they were obsolete before this PR...I sometimes get tempted to clean code, although it is not nice for code review.